templates/helm-values-fluentd-cloudwatch.tpl.yaml

```diff
@@ -141,7 +141,7 @@ fluentdConfig: |
     </parse>
   </source>
 
-  # Drop kubeapi trace,debug and info logs
+  # Drop kubeapi info logs
```

Comment:
If limiting to dropping info, shouldn't you update the pattern bellow accordingly too?

Reply:
It was already the case, just updated the comment to reflect it.

Reply:
ah wait sorry, one sec